main.go

```diff
@@ -30,16 +31,24 @@ var (
 	configFile       = flag.String("config.file", "alertmanager.conf", "Alert Manager configuration file name.")
 	silencesFile     = flag.String("silences.file", "silences.json", "Silence storage file name.")
 	minRefreshPeriod = flag.Duration("alerts.min-refresh-period", 5*time.Minute, "Minimum required alert refresh period before an alert is purged.")
+	pathPrefix       = flag.String("web.path-prefix", "/", "Prefix for all web paths.")
 )
 
 func main() {
 	flag.Parse()
 
+	if !strings.HasPrefix(*pathPrefix, "/") {
+		*pathPrefix = "/" + *pathPrefix
+	}
+	if !strings.HasSuffix(*pathPrefix, "/") {
+		*pathPrefix = *pathPrefix + "/"
+	}
+
 	versionInfoTmpl.Execute(os.Stdout, BuildInfo)
 
 	conf := config.MustLoadFromFile(*configFile)
 
-	silencer := manager.NewSilencer()
+	silencer := manager.NewSilencer(*pathPrefix)
```

Comment:
Confirmed that there is no user of this field at all, and AM still builds when simply removing it from this constructor and the struct type.